src/plugins/stake-plugins/generic/policyAdapters/GlifInfinityPoolAdapter.ts

```diff
@@ -163,7 +163,7 @@ export const makeGlifInfinityPoolAdapter = (policyConfig: StakePolicyConfig<Glif
       // Withdraw liquidity
       txs.push(
         await simpleRampContract.connect(walletSigner).populateTransaction.withdrawF(requestNativeAmount, walletAddress, walletAddress, BigNumber.from(0), {
-          gasLimit: 100000,
+          gasLimit: 10000000,
```

Comment:
Shouldn't the Glif pool give us the recommended gasLimit?

Reply:
Looks like getting rid of the gasLimit worked, so estimateGas did work.